open_lm/utils/llama2openlm.py

```diff
@@ -0,0 +1,151 @@
+"""
+This script converts the weights from LLAMA to OpenLM.
```

Comment:
nit: maybe another open_lm/scripts folder?

Reply:
(this one got merged in https://github.com/mlfoundations/open_lm/pull/45/files so will be removed from here)